src/components/link-preview/link-preview.pw.tsx

```diff
@@ -0,0 +1,258 @@
+    mount,
+    page,
+  }) => {
+    await mount(
+      <LinkPreviewComponentTest image={{ url: urlProp, alt: testCypress }} />
+    );
+
+    await expect(page.locator("img")).toHaveAttribute("src", urlProp);
+
+    await expect(page.locator("img")).toHaveAttribute("alt", testCypress);
+  });
+
+  test("should render Link Preview with url prop", async ({ mount, page }) => {
+    await mount(<LinkPreviewComponentTest url="https://www.foo.com" />);
+
+    await expect(page.getByText("www.foo.com")).toBeAttached();
+  });
+});
+
+test.describe("check functionality for Link Preview component", () => {
+  test("should verify hover color of Link Preview component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkPreviewComponentTest />);
+
+    const previewElement = linkPreview(page);
+    await previewElement.hover();
+
+    await expect(previewElement).toHaveCSS(
+      "background-color",
+      "rgb(204, 214, 219)"
+    );
+  });
+
+  test("should verify border outline color and width of Link Preview on focus", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<LinkPreviewComponentTest />, {
+      hooksConfig: { focusRedesignOptOut: true },
+    });
+
+    await linkPreview(page).focus();
+
+    await checkGoldenOutline(linkPreview(page), "2px");
+  });
+
+  test("should verify border outline color of close icon on focus", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(
+      <LinkPreviewComponentTest as="div" onClose={() => {}} />,
+      {
+        hooksConfig: { focusRedesignOptOut: true },
+      }
+    );
+
+    const closeButton = page.getByLabel("link preview close button");
+
+    await closeButton.focus();
+    await checkGoldenOutline(closeButton);
+  });
+});
+
+test.describe("check events for Link Preview component", () => {
+  test("should call onClose callback when a click event is triggered", async ({
+    mount,
+    page,
+  }) => {
+    let callbackCount = 0;
+    await mount(
+      <LinkPreviewComponentTest
+        as="div"
+        onClose={() => {
+          callbackCount += 1;
+        }}
+      />
+    );
+
+    await linkPreviewCloseIcon(page).click();
+
+    expect(callbackCount).toEqual(1);
+  });
+
+  keysToTrigger.forEach((key) => {
+    test(`should call onClose callback when a keyboard key ${key} event is triggered`, async ({
+      mount,
+      page,
+    }) => {
+      let callbackCount = 0;
+
+      await mount(
+        <LinkPreviewComponentTest
+          as="div"
+          onClose={() => {
+            callbackCount += 1;
+          }}
+        />
+      );
+
+      await page.keyboard.press("Tab");
+      await page.keyboard.down(key);
+
+      await linkPreviewCloseIcon(page).waitFor();
+      expect(callbackCount).toEqual(1);
+    });
+  });
+});
+
+test.describe("Accessibility tests for LinkPreview component", () => {
+  test("should pass accessibilty tests for default story", async ({
```

Comment:
**suggestion: (non blocking)** Might be best to make sure these descriptions do not refer to stories as they are no longer stories.

Something like this should do but up to you what you want to go for 😄 

```suggestion
  test("should pass accessibility tests for default example", async ({
```